CONTRIBUTING.md

```diff
@@ -163,6 +163,8 @@ instructions.
 During development
 + Make sure `CHANGELOG.md` is kept up-to-date with high-level, technical, but user-focused list of changes according to [keepachangelog](https://keepachangelog.com/en/1.0.0/)
 + Bump `UNRELEASED` version in `CHANGELOG.md` according to [semver](https://semver.org/)
++ Ensure `unstable` version of docker images is used in demo
+  - `sed -i -e "s,\(ghcr.io/input-output-hk/hydra-[^:]*\):[^[:space:]]*,\1:unstable," demo/*`
```

Comment:
I think it's more general than this. We should always consider to use the `:unstable` tags or links to `/unstable` documentation etc. during development. While on release we then make sure that everything `unstable` moves to becoming stable or versioned.

Also: this only needs to be done if something becomes incompatible within the context of usage, right?